src/defs.h

```diff
@@ -150,8 +152,17 @@ const uint32_t TEST_FIRST_CONNECTION_FIRST_PACKET_TTL_THRESHOLD_MSEC = 50;
     ":(6553[0-5]|655[0-2][0-9]|65[0-4][0-9]{2}|6[0-4][0-9]{3}|[0-5]?[0-9]{1,4})"                   \
     "(:([a-zA-Z0-9\\.\\-]+|\\[([a-fA-F0-9.:]+)\\]))?[\r\n]"
 
+#ifdef __windows__
+#define UNIX_DOMAIN_SOCKET_FORMAT_REG_EXP                                                          \
+        "^[UuTt]:([A-Za-z]:[\\\\/].*)[\r\n]*"
+#define RESOLVE_ADDR_FORMAT_SOCKET                                                                 \
+        "([A-Za-z]:[\\\\/].*)"
```

Comment:
You also not need to have capture group here because you doesn't use it.

Reply:
both windows and Linux?

Reply:
you're using this regex in the same way on both systems

Reply:
You forgot to remove capture group on Linux

Reply:
Done